### PR TITLE
Themeing related bug fixes and some required changes

### PR DESCRIPTION
**Pull request by ms-hukoyun**
1) wiki page was not using Microsoft footer. Fixed that by overwriting main_django.html since course wiki page is not using main.html but main_django.html
2) In the registration, country error message replaced with A "Country or Region" is required.
3) In the registration changed the Honor Code to Disclosure Notice and fixed the link. Also fixed the error message accordingly.
4) As Jeff requested, made the education and gender  visible again by removing hiding code from css. 
5) Also made the Additional Information section visible again in Account Settings page since we want to allow users to provide or change this info later.
6) When we enable/show additional info, it also showed year of birth field even if it was marked hidden in config. Added css code to hide it.
7) Added css code to hide the full profile message in My Profile page since it was confusing and we don't let users to enter age to promote to full profile.
8) In the sing-on page "Remember Me" was marked as required field even if it was not. Added login.html file to theme package and removed the required field.

I will also submit a PR for oxa-tools to change server-vars.yml for making education and gender optional instead of hidden.
@Microsoft/lex

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            </span>
+          </li>
+        </ol>
+      </div>
+
+      <div class="group group-form group-form-secondary group-form-accountpreferences">
+        <h2 class="sr">${_('Account Preferences')}</h2>
+
+        <ol class="list-input">
+          <li class="field checkbox" id="field-remember">
```
Comment (chenriksson) [inline templates/login.html:200]: Looks like we're only overriding this file to remove required from field-remember?

The checkbox will always have a value (checked/unchecked), so why is this change necessary?

**Review by chenriksson — CHANGES_REQUESTED** [review]

**Review by sdolenc — APPROVED** [review]

File: templates/login.html (hunk 1 above)
Comment (ms-hukoyun) [reply templates/login.html:200]: Actually this was a comment from Jason. As part of accessibility he said this will be confusing for disabled people. As you said it will always have a value of true or false. If you insist I can withdraw this change.

Comment (chenriksson) [reply templates/login.html:200]: No, it's fine. I just missed the comment in the PR description.

**Review by chenriksson — APPROVED** [review]
